html/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/style.css" />
+
+    <link rel='stylesheet'
+    href='https://fonts.googleapis.com/css?family=Poppins' >
+
+      <link rel="stylesheet" 
+      href="path/to/font-awesome/css/font-awesome.min.css">
+
+      <link  rel="stylesheet" 
+      href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css">
+
+    <title>James Port-Folio</title>
+  </head>
+  <body>
+    <header class="header-title">
+        <div class="navbar">
+          <a><p>James</p></a>
+          <nav class="navbar-list">
+            <a href="#" > Porto-folio </a>
+            <a href="#" > About </a>
+            <a href="#" > Contact </a>
+            <a href="#" class="icon"><i class="fa fa-bars"></i></a>
```

Comment:
###⚠️All Optional changes below are very important to do and I mark them optional because the previous reviewer forgot to mention them⚠️

- [ ] [Optional] Please fix the hamburger icon because it doesn't appear on the page.

![image](https://user-images.githubusercontent.com/68796232/170706012-03c66059-f740-4bc0-ac30-05efb4e13ecd.png)